Lib/types.py

```diff
@@ -2,6 +2,7 @@
 Define names for built-in types that aren't directly accessible as a builtin.
 """
 import sys
+import warnings
```

Comment:
Do we need import `warnings`?  I can not see the usage of this module at `types.py`.

Reply:
No :) I forgot about it

Reply:
Fixed